softwareconfiguration/mlplan/src/main/java/de/upb/crc901/mlplan/core/MLPlanBuilder.java

```diff
@@ -157,32 +157,32 @@ public MLPlanBuilder withPreferredNodeEvaluator(INodeEvaluator<TFDNode, Double>
 	}
 
 	@SuppressWarnings("unchecked")
-	public void prepareNodeEvaluatorInFactoryWithData(Instances data) {
-		if (!(hascoFactory instanceof HASCOViaFDAndBestFirstFactory)) {
+	public void prepareNodeEvaluatorInFactoryWithData(final Instances data) {
+		if (!(this.hascoFactory instanceof HASCOViaFDAndBestFirstFactory)) {
 			return;
-//			throw new IllegalStateException("Cannot define a preferred node evaluator if the hasco factory is not a HASCOViaFDAndBestFirstFactory (or a subclass of it)");
+			//			throw new IllegalStateException("Cannot define a preferred node evaluator if the hasco factory is not a HASCOViaFDAndBestFirstFactory (or a subclass of it)");
```

Comment:
here seems to be legacy code in the form of an exception

Reply:
gone